dev-lua/ldoc/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>t@tomoyat1.com</email>
+		<name>Tomoya Tabuchi</name>
+	</maintainer>
```

Comment:
Looks good. Make sure to add the proxy-maint email address to the metadata.xml file as well. Ping me if you need examples.